examples/with-next-ui/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@nextui-org/react": "1.0.0-beta.9",
+    "next": "latest",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@next/bundle-analyzer": "^12.2.0",
+    "@types/node": "18.0.0",
+    "@types/react": "18.0.14",
+    "@types/react-dom": "18.0.5",
+    "add": "^2.0.6",
+    "install": "^0.13.0",
+    "next-compose-plugins": "^2.2.1",
```

Comment:
```suggestion
```

Reply:
Hello, I just joined the package analysis and used this package.

Reply:
Let's remove it to focus the example on NextUI.